poetry.lock

```diff
@@ -1,11 +1,12 @@
-# This file is automatically @generated by Poetry 1.8.5 and should not be changed by hand.
+# This file is automatically @generated by Poetry 2.1.1 and should not be changed by hand.
```

Comment:
i think we had reasons to not upgrade poetry to 2.0.0+.  How do we wnant to handle this?  We're gonna keep fighting back and forth each time we do a pr to this repo

Reply:
I rolled back to poetry 1.8.5. There are still some changes in the poetry lock file - not sure if that's expected.

Reply:
yea because we provide "at least version x" like `tqdm = "^4.67.0"` for a lot of our packages, it will auto update to the latest every time we do a `poetry update`.  I actually wasnt thinking you should come down to 1.8.5, i was thinking we might wanna go up BUT would love for this to be added to the readme updates you're doing around our set up documentation. Thanks!